deploy_to_heroku.md

```diff
@@ -0,0 +1,92 @@
+# Deploy to Heroku
+
+
+## Process
+
+1. Ensure you have an active and valid Heroku account
+2. Install the Heroku CLI
+   - See [Getting Started on Heroku with Python | Heroku Dev
+     Center][herokupython]
+   - Note autocomplete instructions
+3. Prepare the app:
+   1. Clone the repository using git and change to its directory
+   2. Create the app:
+        ```shell
+        heroku create --team=TEAMNAME
+        ```
+      - Allow Heroku to set a random name
+      - Specify the team (omit if not applicable)
+      - If you want a specific Heroku PostgreSQL plan, it's easiest to specify
+        it here (ex. `--addons=heroku-postgresql:standard-0`)
+3. Mange confign variables (this can be done using the Heroku Dashboard too)
```

Comment:
Looks great! ✨ Thanks to you @TimidRobot for write down the instructions.

Just a little typo here I think: "Manage config variables..."